lib/publiccloud/ec2.pm

```diff
@@ -177,6 +177,10 @@ sub img_proof {
 }
 
 sub cleanup {
+    my ($self) = @_;
```

Comment:
for `gce::cleanup()` we are calling `SUPER::cleanup()` and here we don't any good reason for this ? generally I am against having this empty calls . let's keep only pieces which actually doing something . IMO it will be frustrating for person who will try to read and understand this code to jump between several files just to figure out that there is only empty stubs calling other empty stubs ...